astropy/stats/tests/test_funcs.py

```diff
@@ -337,7 +336,10 @@ def test_mad_std_scalar_return():
         with catch_warnings():
             rslt = funcs.mad_std(data)
             assert np.isscalar(rslt)
-            assert not np.isnan(rslt)
+            try:
+                assert not np.isnan(rslt)
+            except AssertionError:
+                pytest.xfail('See #5232')
```

Comment:
That PR discussion is really long; maybe add a link to a specific comment that summarizes the expected fail?  I don't remember if this particular test is expected to fail only on certain numpy versions or what...

Reply:
I wasn't involved in that discussion and can't follow it completely (it *is* very long). Is https://github.com/astropy/astropy/pull/5232#issuecomment-239181714 close enough? It does seem weird that this test would only `xpass` in Numpy 1.11 and nothing else.

Reply:
...it would probably take me half an hour to figure it out reading back through that.  Oh well, let's just leave the comment here - at least it is theoretically possible to reconstruct my mental state at the time...

Reply:
Okay then. :wink:

Reply:
If that comment is right, this problem will auto-disappear in 3.1, since we'll only support numpy >=1.13

That also suggests, though, this should really be a test on numpy version. I don't really want to hold up this supposedly simple PR, but could you add a comment that might well be resolved in later numpy? Please include something like `NUMPY_LT_1_13`?? in the comment, so that we'll find it!